README_zh-CN.md

```diff
@@ -219,7 +243,7 @@ MMClassification 是一款由不同学校和公司共同贡献的开源项目。
 - [MMCV](https://github.com/open-mmlab/mmcv): OpenMMLab 计算机视觉基础库
 - [MIM](https://github.com/open-mmlab/mim): MIM 是 OpenMMlab 项目、算法、模型的统一入口
 - [MMEval](https://github.com/open-mmlab/mmeval): 统一开放的跨框架算法评测库
-- [MMClassification](https://github.com/open-mmlab/mmclassification): OpenMMLab 图像分类工具箱
+- [MMPreTrain](https://github.com/open-mmlab/mmpretrain): OpenMMLab 视觉预训练工具箱
```

Comment:
Should we delete mmselfsup to avoid it taking away some of the traffic?

Reply:
maybe keep it for sometime, we have added mmpretrain's link in mmselfsup